### ISSUE TITLE
Assertion fail during editing translation(wx/string.h(1536): assert "!empty()" failed in Last(): wxString: index out of bounds)

### ISSUE THREAD

**Issue by Vdragon** [vslavik/poedit#237]
Crash during regular translation editing, using a Chinese input method called [HIME](http://github.com/hime-ime/hime).

It seems that at that time I was committing text from input method's pre-edit buffer to the application translation field, however I've done the same thing several times without producing the crash.

**UPDATE:** I've reproduced this issue again but still don't have an accurate reproducible way, I'm pretty sure this is input-method related.

## Application Crash Information
```
ASSERT INFO:
/usr/include/wx-3.0/wx/string.h(1536): assert "!empty()" failed in Last(): wxString: index out of bounds

BACKTRACE:
[1] wxString wxString::Format<wxString, wxString>(wxFormatString const&, wxString, wxString)
[2] std::__weak_count<(__gnu_cxx::_Lock_policy)2>::operator=(std::__shared_count<(__gnu_cxx::_Lock_policy)2> const&)
[3] wxAppConsoleBase::CallEventHandler(wxEvtHandler*, wxEventFunctor&, wxEvent&) const
[4] wxEvtHandler::ProcessEventIfMatchesId(wxEventTableEntryBase const&, wxEvtHandler*, wxEvent&)
[5] wxEvtHandler::SearchDynamicEventTable(wxEvent&)
[6] wxEvtHandler::TryHereOnly(wxEvent&)
[7] wxEvtHandler::DoTryChain(wxEvent&)
[8] wxEvtHandler::ProcessEvent(wxEvent&)
[9] wxEvtHandler::SafelyProcessEvent(wxEvent&)
[10] wxTextEntryBase::SendTextUpdatedEvent(wxWindow*)
[11] g_closure_invoke
[12] g_signal_emit_valist
[13] g_signal_emit
[14] g_closure_invoke
[15] g_signal_emit_valist
[16] g_signal_emit
[17] gtk_text_buffer_delete_interactive
[18] gtk_text_buffer_delete_selection
[19] g_cclosure_marshal_VOID__STRINGv
[20] g_signal_emit_valist
[21] g_signal_emit_by_name
[22] g_cclosure_marshal_VOID__STRINGv
[23] g_signal_emit_valist
[24] g_signal_emit_by_name
[25] g_closure_invoke
[26] g_signal_emit_valist
[27] g_signal_emit
[28] gtk_window_propagate_key_event
[29] g_closure_invoke
[30] g_signal_emit_valist
[31] g_signal_emit
[32] gtk_propagate_event
[33] gtk_main_do_event
[34] g_main_context_dispatch
[35] g_main_loop_run
[36] gtk_main
[37] wxGUIEventLoop::DoRun()
[38] wxEventLoopBase::Run()
[39] wxAppConsoleBase::MainLoop()
[40] wxEntry(int&, wchar_t**)
[41] std::_Sp_counted_base<(__gnu_cxx::_Lock_policy)2>::_M_destroy()
[42] __libc_start_main
```

### Operating System
Ubuntu 15.10 AMD64

### Poedit
1.8.4-1 from Ubuntu 15.10 software source

### WxWidget
3.0.2+dfsg-1.2 from Ubuntu 15.10 software source

### Locale
```
LANG=zh_TW.UTF-8
LANGUAGE=zh_TW:zh_HK:zh_CN:en_US:en
LC_CTYPE="zh_TW.UTF-8"
LC_NUMERIC=zh_TW.UTF-8
LC_TIME=zh_TW.UTF-8
LC_COLLATE="zh_TW.UTF-8"
LC_MONETARY=zh_TW.UTF-8
LC_MESSAGES="zh_TW.UTF-8"
LC_PAPER=zh_TW.UTF-8
LC_NAME=zh_TW.UTF-8
LC_ADDRESS=zh_TW.UTF-8
LC_TELEPHONE=zh_TW.UTF-8
LC_MEASUREMENT=zh_TW.UTF-8
LC_IDENTIFICATION=zh_TW.UTF-8
LC_ALL=
```

### HIME Input Method
Self built version with commit 900d88868

**Comment by vslavik** [comment]
When any form of backtrace is involved in any way, *always* make sure debug symbols are present. That's why Ubuntu has *-dbg packages and why its guide mentions them. Backtrace without debug symbols is mostly useless; this one, for example, is 100% certain to be lying (the 1-2-3 sequence of calls is impossible).

This looks like something that could be fixable even without reproduction ("I've seen it" isn't what "reproduce" means; step by step instructions on how to reliably repeat it are), if only the backtrace was worth anything.

Also please *always* try using the latest version (1.8.4 isn't).

**Comment by Vdragon** [comment]
The following is the backtrace attempted using GDB(The crash dialog provided by WxWidget didn't provide any deeper details even I have installed most debugging symbols, so I manually send a SIGARBT to poedit process in order to force gdb prompt showing up)

```
#0  0x00007ffff25a08dd in poll () at ../sysdeps/unix/syscall-template.S:81
#1  0x00007ffff45c71ec in g_main_context_iterate (priority=2147483647, n_fds=4, fds=0xb8bfe0, timeout=<optimized out>, context=0xafbb90)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./glib/gmain.c:4135
#2  0x00007ffff45c71ec in g_main_context_iterate (context=0xafbb90, block=block@entry=1, dispatch=dispatch@entry=1, self=<optimized out>)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./glib/gmain.c:3835
#3  0x00007ffff45c7572 in g_main_loop_run (loop=0x16450d0)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./glib/gmain.c:4034
#4  0x00007ffff33e7ee3 in IA__gtk_dialog_run (dialog=0xf08030 [GtkAssertDialog])
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtkdialog.c:1094
#5  0x00007ffff6c51a6b in wxGUIAppTraits::ShowAssertDialog(wxString const&) (this=<optimized out>, msg=...) at ../src/gtk/utilsgtk.cpp:316
#6  0x00007ffff634ab72 in ShowAssertDialog(wxString const&, int, wxString const&, wxString const&, wxString const&, wxAppTraits*) (file=..., line=line@entry=1536, func=..., cond=..., msgUser=..., traits=traits@entry=0xb81940)
    at ../src/common/appbase.cpp:1319
#7  0x00007ffff634eb97 in wxAppConsoleBase::OnAssertFailure(wchar_t const*, int, wchar_t const*, wchar_t const*, wchar_t const*) (this=this@entry=0xad10f0, file=<optimized out>, line=1536, func=<optimized out>, cond=<optimized out>, msg=<optimized out>) at ../src/common/appbase.cpp:798
#8  0x00007ffff6c0ffa0 in wxApp::OnAssertFailure(wchar_t const*, int, wchar_t const*, wchar_t const*, wchar_t const*) (this=0xad10f0, file=<optimized out>, line=<optimized out>, func=<optimized out>, cond=<optimized out>, msg=<optimized out>)
    at ../src/gtk/app.cpp:507
#9  0x00007ffff634f262 in wxDefaultAssertHandler(wxString const&, int, wxString c---Type <return> to continue, or q <return> to quit---
onst&, wxString const&, wxString const&) (file=..., line=1536, func=..., cond=..., msg=...) at ../src/common/appbase.cpp:1110
#10 0x00007ffff634e0fa in wxOnAssert(char const*, int, char const*, char const*, wchar_t const*) (file=0x6ae8e0 "/usr/include/wx-3.0/wx/string.h", line=1536, func=<optimized out>, cond=<optimized out>, msg=<optimized out>)
    at ../src/common/appbase.cpp:1195
#11 0x000000000048c554 in wxString::Last() (this=0x7fffffffb210)
    at /usr/include/wx-3.0/wx/string.h:1536
#12 0x00000000005d553e in TranslationTextCtrl::OnText(wxCommandEvent&) (this=
    0x1507400, e=...) at text_control.cpp:735
#13 0x00007ffff63472de in wxAppConsoleBase::CallEventHandler(wxEvtHandler*, wxEventFunctor&, wxEvent&) const (this=0xad10f0, handler=0x1507400, functor=..., event=...) at ../src/common/appbase.cpp:623
#14 0x00007ffff64cd537 in wxEvtHandler::ProcessEventIfMatchesId(wxEventTableEntryBase const&, wxEvtHandler*, wxEvent&) (entry=..., handler=<optimized out>, event=...) at ../src/common/event.cpp:1392
#15 0x00007ffff64cd91e in wxEvtHandler::SearchDynamicEventTable(wxEvent&) (this=0x1507400, event=...) at ../src/common/event.cpp:1751
#16 0x00007ffff64cd9ae in wxEvtHandler::TryHereOnly(wxEvent&) (this=0x1507400, event=...) at ../src/common/event.cpp:1585
#17 0x00007ffff64cd7e3 in wxEvtHandler::DoTryChain(wxEvent&) (this=<optimized out>, event=...) at ../src/common/event.cpp:1554
#18 0x00007ffff64cdad5 in wxEvtHandler::ProcessEvent(wxEvent&) (this=0x1378f80, event=...) at ../src/common/event.cpp:1495
#19 0x00007ffff64cd837 in wxEvtHandler::SafelyProcessEvent(wxEvent&) (this=<optimized out>, event=...) at ../src/common/event.cpp:1613
---Type <return> to continue, or q <return> to quit---
#20 0x00007ffff6e4254c in wxWindowBase::HandleWindowEvent(wxEvent&) const (this=this@entry=0x1507400, event=...) at ../src/common/wincmn.cpp:1525
#21 0x00007ffff6e2ce8f in wxTextEntryBase::SendTextUpdatedEvent(wxWindow*) (win=0x1507400) at ../src/common/textentrycmn.cpp:378
#25 0x00007ffff48b712f in <emit signal ??? on instance 0x12b1e30 [GtkTextBuffer]> (instance=instance@entry=0x12b1e30, signal_id=<optimized out>, detail=detail@entry=0) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3439
    #22 0x00007ffff489c015 in g_closure_invoke (closure=0x13763c0, return_value=return_value@entry=0x0, n_param_values=1, param_values=param_values@entry=0x7fffffffb5f0, invocation_hint=invocation_hint@entry=0x7fffffffb570)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gclosure.c:801
    #23 0x00007ffff48ae061 in signal_emit_unlocked_R (node=node@entry=0x1138c50, detail=detail@entry=0, instance=instance@entry=0x12b1e30, emission_return=emission_return@entry=0x0, instance_and_params=instance_and_params@entry=0x7fffffffb5f0) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3627
    #24 0x00007ffff48b6dfc in g_signal_emit_valist (instance=<optimized out>, signal_id=<optimized out>, detail=<optimized out>, var_args=var_args@entry=0x7fffffffb7a0) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3383
#26 0x00007ffff34fa589 in gtk_text_buffer_real_delete_range (buffer=0x12b1e30 [GtkTextBuffer], start=0x7fffffffbdf0, end=0x7fffffffbd50)
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtktextbuffer.c:1595
#30 0x00007ffff48b712f in <emit signal ??? on instance 0x12b1e30 [GtkTextBuffer]> (instance=instance@entry=0x12b1e30, signal_id=<optimized out>, detail=detail@entry=0) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3439
    #27 0x00007ffff489c015 in g_closure_invoke (closure=closure@entry=0x12f7f00, return_value=return_value@entry=0x0, n_param_values=3, param_values=param_values@---Type <return> to continue, or q <return> to quit---
entry=0x7fffffffba60, invocation_hint=invocation_hint@entry=0x7fffffffb9e0)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gclosure.c:801
    #28 0x00007ffff48ae60e in signal_emit_unlocked_R (node=node@entry=0x12f7f50, detail=detail@entry=0, instance=instance@entry=0x12b1e30, emission_return=emission_return@entry=0x0, instance_and_params=instance_and_params@entry=0x7fffffffba60) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3665
    #29 0x00007ffff48b6dfc in g_signal_emit_valist (instance=<optimized out>, signal_id=<optimized out>, detail=<optimized out>, var_args=var_args@entry=0x7fffffffbc40) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3383
#31 0x00007ffff34f4c97 in gtk_text_buffer_emit_delete (buffer=buffer@entry=0x12b1e30 [GtkTextBuffer], start=start@entry=0x7fffffffbdf0, end=end@entry=0x7fffffffbd50) at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtktextbuffer.c:1613
#32 0x00007ffff34f98ab in IA__gtk_text_buffer_delete_interactive (buffer=buffer@entry=0x12b1e30 [GtkTextBuffer], start_iter=start_iter@entry=0x7fffffffbe90, end_iter=end_iter@entry=0x7fffffffbee0, default_editable=default_editable@entry=1)
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtktextbuffer.c:1725
#33 0x00007ffff34f9c81 in IA__gtk_text_buffer_delete_selection (buffer=0x12b1e30 [GtkTextBuffer], interactive=interactive@entry=1, default_editable=default_editable@entry=1) at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtktextbuffer.c:3819
#34 0x00007ffff351d805 in gtk_text_view_commit_text (text_view=0x12afc60 [GtkTextView], str=0x1622620 "壓")
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtktextview.c:7455
#35 0x00007ffff489eaca in g_cclosure_marshal_VOID__STRINGv (closure=0x1507ba0, return_value=<optimized out>, instance=<optimized out>, args=<optimized out>, marshal_data=<optimized out>, n_params=<optimized out>, param_types=0xf25ae0)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gmarshal.c:1794
---Type <return> to continue, or q <return> to quit---
#36 0x00007ffff489c244 in _g_closure_invoke_va (closure=closure@entry=0x1507ba0, return_value=return_value@entry=0x0, instance=instance@entry=0x113d030, args=args@entry=0x7fffffffc2b8, n_params=<optimized out>, param_types=0xf25ae0)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gclosure.c:864
#37 0x00007ffff48b6a46 in g_signal_emit_valist (instance=instance@entry=0x113d030, signal_id=signal_id@entry=248, detail=detail@entry=0, var_args=var_args@entry=0x7fffffffc2b8) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3292
#38 0x00007ffff48b75f5 in g_signal_emit_by_name (instance=0x113d030, detailed_signal=0x7ffff36019da "commit")
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3479
#39 0x00007ffff489eaca in g_cclosure_marshal_VOID__STRINGv (closure=0x1509c90, return_value=<optimized out>, instance=<optimized out>, args=<optimized out>, marshal_data=<optimized out>, n_params=<optimized out>, param_types=0xf25ae0)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gmarshal.c:1794
#40 0x00007ffff489c244 in _g_closure_invoke_va (closure=closure@entry=0x1509c90, return_value=return_value@entry=0x0, instance=instance@entry=0x13d6720, args=args@entry=0x7fffffffc6a8, n_params=<optimized out>, param_types=0xf25ae0)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gclosure.c:864
#41 0x00007ffff48b6a46 in g_signal_emit_valist (instance=instance@entry=0x13d6720, signal_id=signal_id@entry=248, detail=detail@entry=0, var_args=var_args@entry=0x7fffffffc6a8) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3292
#42 0x00007ffff48b75f5 in g_signal_emit_by_name (instance=0x13d6720, detailed_signal=0x7fffdedd3b9c "commit")
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3479
#43 0x00007fffdedd3516 in gtk_im_context_hime_filter_keypress (context=0x13d6720 [GtkIMContextHIME], event=0x15a7470) at gtkimcontexthime.c:471
---Type <return> to continue, or q <return> to quit---
#44 0x00007ffff3449e03 in gtk_im_multicontext_filter_keypress (context=0x113d030 [GtkIMMulticontext], event=0x15a7470)
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtkimmulticontext.c:333
#45 0x00007ffff6c5f682 in gtk_window_key_press_callback(GtkWidget*, GdkEventKey*, wxWindow*) (gdk_event=0x15a7470, win=0x1507400) at ../src/gtk/window.cpp:1012
#50 0x00007ffff48b712f in <emit signal ??? on instance 0x12afc60 [GtkTextView]> (instance=instance@entry=0x12afc60, signal_id=<optimized out>, detail=detail@entry=0) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3439
    #46 0x00007ffff3460e5c in _gtk_marshal_BOOLEAN__BOXED (closure=0x150b920, return_value=0x7fffffffd190, n_param_values=<optimized out>, param_values=0x7fffffffd1f0, invocation_hint=<optimized out>, marshal_data=<optimized out>)
    at gtkmarshalers.c:86
    #47 0x00007ffff489c015 in g_closure_invoke (closure=0x150b920, return_value=return_value@entry=0x7fffffffd190, n_param_values=2, param_values=param_values@entry=0x7fffffffd1f0, invocation_hint=invocation_hint@entry=0x7fffffffd170)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gclosure.c:801
    #48 0x00007ffff48ae061 in signal_emit_unlocked_R (node=node@entry=0xb2edd0, detail=detail@entry=0, instance=instance@entry=0x12afc60, emission_return=emission_return@entry=0x7fffffffd300, instance_and_params=instance_and_params@entry=0x7fffffffd1f0) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3627
    #49 0x00007ffff48b6899 in g_signal_emit_valist (instance=<optimized out>, signal_id=<optimized out>, detail=<optimized out>, var_args=var_args@entry=0x7fffffffd3b0) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3393
#51 0x00007ffff3578cdc in gtk_widget_event_internal (widget=widget@entry=0x12afc60 [GtkTextView], event=event@entry=0x15a7470)
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtkwidget.c:5017
---Type <return> to continue, or q <return> to quit---
#52 0x00007ffff3578fb7 in IA__gtk_widget_event (widget=widget@entry=0x12afc60 [GtkTextView], event=event@entry=0x15a7470)
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtkwidget.c:4814
#53 0x00007ffff358c79d in IA__gtk_window_propagate_key_event (window=window@entry=0xc74200 [GtkWindow], event=event@entry=0x15a7470)
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtkwindow.c:5199
#54 0x00007ffff6c4bce8 in wxgtk_tlw_key_press_event(GtkWidget*, GdkEventKey*) (widget=widget@entry=0xc74200 [GtkWindow], event=0x15a7470)
    at ../src/gtk/toplevel.cpp:243
#59 0x00007ffff48b712f in <emit signal ??? on instance 0xc74200 [GtkWindow]> (instance=instance@entry=0xc74200, signal_id=<optimized out>, detail=detail@entry=0)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3439
    #55 0x00007ffff3460e5c in _gtk_marshal_BOOLEAN__BOXED (closure=0xc88df0, return_value=0x7fffffffd6b0, n_param_values=<optimized out>, param_values=0x7fffffffd710, invocation_hint=<optimized out>, marshal_data=<optimized out>)
    at gtkmarshalers.c:86
    #56 0x00007ffff489c015 in g_closure_invoke (closure=0xc88df0, return_value=return_value@entry=0x7fffffffd6b0, n_param_values=2, param_values=param_values@entry=0x7fffffffd710, invocation_hint=invocation_hint@entry=0x7fffffffd690)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gclosure.c:801
    #57 0x00007ffff48ae061 in signal_emit_unlocked_R (node=node@entry=0xb2edd0, detail=detail@entry=0, instance=instance@entry=0xc74200, emission_return=emission_return@entry=0x7fffffffd820, instance_and_params=instance_and_params@entry=0x7fffffffd710) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3627
    #58 0x00007ffff48b6899 in g_signal_emit_valist (instance=<optimized out>, signal_id=<optimized out>, detail=<optimized out>, var_args=var_args@entry=0x7ffffff---Type <return> to continue, or q <return> to quit---
fd8d0) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./gobject/gsignal.c:3393
#60 0x00007ffff3578cdc in gtk_widget_event_internal (widget=widget@entry=0xc74200 [GtkWindow], event=event@entry=0x15a7470)
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtkwidget.c:5017
#61 0x00007ffff3578fb7 in IA__gtk_widget_event (widget=widget@entry=0xc74200 [GtkWindow], event=event@entry=0x15a7470)
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtkwidget.c:4814
#62 0x00007ffff345f6d7 in IA__gtk_propagate_event (widget=0xc74200 [GtkWindow], event=0x15a7470) at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtkmain.c:2494
#63 0x00007ffff345f9ab in IA__gtk_main_do_event (event=0x15a7470)
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtkmain.c:1710
#64 0x00007ffff1d5dc0c in gdk_event_dispatch (source=<optimized out>, callback=<optimized out>, user_data=<optimized out>)
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gdk/x11/gdkevents-x11.c:2425
#65 0x00007ffff45c6ff7 in g_main_context_dispatch (context=0xafbb90)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./glib/gmain.c:3154
#66 0x00007ffff45c6ff7 in g_main_context_dispatch (context=context@entry=0xafbb90) at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./glib/gmain.c:3769
#67 0x00007ffff45c7250 in g_main_context_iterate (context=0xafbb90, block=block@entry=1, dispatch=dispatch@entry=1, self=<optimized out>)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./glib/gmain.c:3840
#68 0x00007ffff45c7572 in g_main_loop_run (loop=0xb1c2e0)
    at /build/glib2.0-ajuDY6/glib2.0-2.46.1/./glib/gmain.c:4034
#69 0x00007ffff345ea07 in IA__gtk_main ()
    at /build/gtk+2.0-xmBAGF/gtk+2.0-2.24.28/gtk/gtkmain.c:1282
#70 0x00007ffff6c2c055 in wxGUIEventLoop::DoRun() (this=0xade6f0)
---Type <return> to continue, or q <return> to quit---
    at ../src/gtk/evtloop.cpp:65
#71 0x00007ffff63892b3 in wxEventLoopBase::Run() (this=0xade6f0)
    at ../src/common/evtloopcmn.cpp:78
#72 0x00007ffff634e746 in wxAppConsoleBase::MainLoop() (this=0xad10f0)
    at ../src/common/appbase.cpp:334
#73 0x00007ffff63dae80 in wxEntry(int&, wchar_t**) (argc=<optimized out>, argv=<optimized out>) at ../src/common/init.cpp:495
#74 0x000000000049b101 in main(int, char**) (argc=1, argv=0x7fffffffddb8)
    at edapp.cpp:253
```

I'm trying to reproduce it on 1.8.7 but I'm still failed to build it by the time.

**Comment by vslavik** [comment]
> I'm trying to reproduce it on 1.8.7 but I'm still failed to build it by the time.

Let me try fixing it based on backtrack only (thanks!), so that you don't waste any more time on it needlessly – it looks clear enough at first glance.

**Comment by Vdragon** [comment]
The issue on building the latest version is filed at #238

**Comment by vslavik** [comment]
Fixed in 1.8.5 (f3eb45de37175fe4b4cc4f2c6ff11b107192b26f) — the same bug as #203.

(This is exactly why "always try the latest version first" is a meaningful advice…)
